fire_atlas_editor/lib/screens/editor_screen/widgets/selection_canvas/canvas_sprite.dart

```diff
@@ -130,11 +126,14 @@ class _CanvasSpritePainer extends CustomPainter {
 
     // Selection
     if (_selectionStart != null && _selectionEnd != null) {
-      final size = _selectionEnd - _selectionStart + Offset(1, 1);
+      final _start = _selectionStart!;
+      final _end = _selectionEnd!;
```

Comment:
These could be declared before the if and then the null check done on these and then the ! wouldn't be needed inside the if right?

Reply:
Yes! Changed